hibernate-core/src/main/java/org/hibernate/sql/Delete.java

```diff
@@ -13,7 +13,7 @@
 import org.hibernate.sql.ast.spi.ParameterMarkerStrategy;
 
 /**
- * A SQL {@code DELETE} statement.
+ * An SQL {@code DELETE} statement.
```

Comment:
Meh. Most people pronounce this "A sequel", not "An S-Q-L". I don't think we need to police this.